typescript/phoenix-sdk/config.json

```diff
@@ -52,7 +52,7 @@
         "logoUri": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/2FPyTwcZLUg1MDrwsyoP4D6s1tM7hAkHYRjkNb5w6Pxk/logo.png"
       }
     ],
-    "markets": []
+    "markets": ["14CAwu3LiBBk5fcHGdTsFyVxDwvpgFiSfDwgPJxECcE5"]
```

Comment:
do we need to add tokens here? "BASE" and "QUOTE"?

also, let's get rid of the sollet tokens